scripts/migrate_schemas_to_db.py

```
@@ -0,0 +1,152 @@
+    for file_path in schemas_path.glob("*.json"):
+        try:
+            with open(file_path, 'r') as f:
+                schema = json.load(f)
+                if 'name' in schema:
+                    schemas.append(schema)
+                    print(f"Loaded schema '{schema['name']}' from {file_path.name}")
+                else:
+                    print(f"Warning: Schema file '{file_path.name}' missing 'name' field, skipping")
+        except json.JSONDecodeError as e:
+            print(f"Error: Failed to parse {file_path.name}: {e}")
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)